cmake/ValhallaPkgConfig.cmake

```diff
@@ -23,18 +23,9 @@ function(configure_valhalla_pc)
     list(APPEND REQUIRES protobuf)
   endif()
   
-  if(ENABLE_DATA_TOOLS)
-    list(APPEND REQUIRES spatialite sqlite3 luajit geos openssl)
-  endif()
-  if(ENABLE_HTTP)
-    list(APPEND REQUIRES libcurl)
-  endif()
```

Comment:
That's too much now;) except for spatialite and openssl, they're exposed in our headers and needed to resolve libvalhalla dependencies.

Reply:
I should return to a computer for review😄